rfml.go

```diff
@@ -326,34 +326,69 @@ func uploadSingleRFMLFile(filePath string) error {
 	if err != nil {
 		return fileParseError{filePath, err}
 	}
+	parsedTest.RFMLPath = filePath
 
 	// Check if the test already exists in RF so we can decide between updating and creating new one
 	mappings, err := api.GetRFMLIDs()
 	if err != nil {
 		return err
 	}
-	err = parsedTest.PrepareToUploadFromRFML(mappings)
-	if err != nil {
-		return err
-	}
-	if parsedTest.TestID == 0 {
-		// Create new test
+
+	testID, ok := mappings.MapRFMLIDtoID()[parsedTest.RFMLID]
+	if ok {
+		parsedTest.TestID = testID
+	} else {
+		// Create an empty test
```

Comment:
Do we need to go through empty test route if we are uploading just a single file?